agent/proto/agentpb/agent.proto

```diff
@@ -126,6 +126,14 @@ message TimeResponse {
   Timestamp timestamp = 1;
 }
 
+// TimelineRequest is a request to get the cluster status timeline
+message TimelineRequest {}
+
+// TimelineResponse contains the cluster status timeline events
+message TimelineResponse {
+  repeated string timeline = 1;
```

Comment:
I think we should return a structured response, as a list of timeline events for example.